slide to frontpage

lipefsa opened the pull request:
slider adicionado
cta corrigido
banner ajustado

lipefsa committed:
cta updated

lipefsa committed:
swiper funcionando - ajeitar css

lipefsa committed:
slide funcionando

lipefsa committed:
titulo do card ajustado

lipefsa committed:
imagem ajustada

JoaoCFN commented on .gitignore:
```diff
@@ -1,2 +1,4 @@
-.env
\ No newline at end of file
+.env
+.todo
```
@lipefsa Recomendo remover esse arquivo .todo do gitignore. Ele na verdade não deveria subir com essa PR.

JoaoCFN commented on pages/landing_page/landing_page.js:
```diff
@@ -0,0 +1,127 @@
+const apiPath = "/house_of_barber/api";
+
```
@lipefsa Recomendo remover os espaços vazios remanescentes nesse arquivo. Ficou muito espaço fantasma.

JoaoCFN commented on pages/landing_page/content/landing_page_content.php:
```diff
@@ -25,33 +23,45 @@ class="logo"
 </section>
 
+<section class="swiper-section">
+<div class="swiper mySwiper">
```
@lipefsa Ajustar formatação do código nesse trecho

JoaoCFN commented on pages/landing_page/landing_page.js:
```diff
@@ -0,0 +1,127 @@
+
+                    cardsBarbeariaWrapper.innerHTML += `
+                        <div class='swiper-slide'>
+                            <div class='card area-cliente-card'>
```
@lipefsa Recomendo trocar a nomenclatura aqui. `area-cliente-card` não faz sentido nesse contexto. Busca colocar um nome que faça sentido para a landing page.

JoaoCFN commented on pages/landing_page/landing_page.css:
```diff
@@ -15,46 +15,141 @@
+    
+}
+
+.area-cliente .card-title {
```
@lipefsa Mesmo ponto sobre a nomenclatura aqui. `area-cliente` não é um bom nome para uma seção dentro da landing page.

JoaoCFN commented on assets/images/login.png:
@lipefsa Vê se tu consegue achar uma imagem mais próxima da tela de login atual para a substituição. Não vejo essas imagens conversando muito com a identidade visual do sistema, sabe?

JoaoCFN commented on assets/images/login2.png:
@lipefsa Mesmo ponto sobre as imagens aqui.

JoaoCFN reviewed (changes requested):
@lipefsa 
Descrevi alguns ajustes que devem ser feitos na PR nos comentários


Encontrei também um bug. Devido a ausência do token de validação, o carregamento do slide não funcionou corretamente:

![image](https://github.com/lipefsa/house_of_barber/assets/58020444/a038b97d-c524-402f-b0ba-eff1b89bf496)

Outro ponto importante, é que o container do menu e o banner foi limitado. Esse comportamento não deve ocorrer.

**Situação correta:**

![image](https://github.com/lipefsa/house_of_barber/assets/58020444/1e919596-dde7-4acc-8780-b324519e0ea7)

**Situação da sua branch:**

![image](https://github.com/lipefsa/house_of_barber/assets/58020444/4f7a42d9-8e13-4db4-9a23-b449f24e73ce)

lipefsa committed:
1. gitignore resolvido
2. indentação corrigida
3. nomes de classes e variaveis ajustadas
4. tela de login atualizada
5. banner landing page corrigido

lipefsa committed:
cards na landing page não precisam autenticar

lipefsa commented:
corrigido

Commit:
fix: ajustando imagem da tela de logn

Commit:
fix: realizando melhorias no carrossel

JoaoCFN committed:
Merge branch 'main' into lipebranch
